en/messages.json

```diff
@@ -351,5 +351,66 @@
   },
   "integrateContainers": {
     "message": "Integrate your Containers with Mozilla VPN"
+  },
+  "backupFailure": {
+    "message": "Something goes wrong, backup failed: $errorMessage$",
+    "description": "(Options menu) When exporting containers as a file failed",
+    "placeholders": {
+      "errorMessage": {
+        "content": "$1"
+      }
+    }
+  },
+  "containersRestored": {
+    "message": "$restoredCount$ containers restored.",
+    "description": "(Options menu) When importing containers from a file succeed",
+    "placeholders": {
+      "restoredCount": {
+        "content": "$1",
+        "example": "12"
+      }
+    }
+  },
+  "containersPartiallyRestored": {
+    "message": "$restoredCount$ containers restored, but some lost their site association : $incompleteList$.",
+    "description": "(Options menu) When importing containers from a file succeed, but some containers associations couldn't be restored",
+    "placeholders": {
+      "restoredCount": {
+        "content": "$1",
+        "example": "12"
+      },
+      "incompleteList": {
+        "content": "$1",
+        "example": "MyFacebookContainer, MyTwitterContainer"
+      }
+    }
+  },
+  "containersRestorationFailed": {
+    "message": "The file is corrupted, or isn't a container backup file.",
+    "description": "(Options menu) When importing containers from a file failed (most likely due to a malformed file)"
+  },
+  "backup": {
+    "message": "Backup",
+    "description": "(Options menu) Backup to/from a file header"
+  },
+  "restoreLegend": {
+    "message": "Restore",
+    "description": "(Options menu) Restore containers from a file group header"
+  },
+  "warningConfigOverride": {
+    "message": "WARNING ! This operation will erase current configuration with the imported one. All cookies associated with existing containers will be discarded.",
+    "description": "(Options menu) Warns the user that old containers will be deleted along with their associated cookies"
+  },
+  "saveLegend": {
+    "message": "Save",
+    "description": "(Options menu) Save containers to a file group header"
+  },
+  "saveButton": {
+    "message": "Save",
+    "description": "(Options menu) Validate saving containers to a file"
+  },
+  "noteWontBackupCookies": {
+    "message": "Backups only containers names, icons, colors and site assosiations, but NOT containers' cookies.",
```

Comment:
Besides the typo (associations), this looks off as well. Either "backups" should be a verb, and it's misspelled, or the string is missing a verb.